static/app/components/suggestProjectCTA.tsx

```diff
@@ -190,16 +189,14 @@ class SuggestProjectCTA extends Component<Props, State> {
 
   renderCTA() {
     return (
-      <Alert type="info">
-        <Content>
-          <span>
-            {tct(
-              'We have a sneaking suspicion you have a mobile app that doesn’t use Sentry. [link:Start Monitoring]',
-              {link: <a onClick={this.openModal} />}
-            )}
-          </span>
-          <StyledIconClose onClick={this.handleCTAClose} />
-        </Content>
+      <Alert
+        type="info"
+        trailingItems={<StyledIconClose onClick={this.handleCTAClose} />}
+      >
+        {tct(
+          'We have a sneaking suspicion you have a mobile app that doesn’t use Sentry. [link:Start Monitoring]',
+          {link: <a onClick={this.openModal} />}
```

Comment:
should we use a Button with a priority link here?

Reply:
hmm here I think we can use a normal button and move it to `trailingItems`, https://github.com/getsentry/sentry/pull/32781/commits/1c957318f6ad4306aca8eabd8259e0b95f10c1c0:
<img width="931" alt="Screen Shot 2022-03-24 at 10 37 05 AM" src="https://user-images.githubusercontent.com/44172267/159976861-a4ba499d-cb01-4332-8540-8460a00bcb9e.png">